### PR TITLE
release: release 2.1 version.

### PR DESCRIPTION
**Pull request by moonming**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: rockspec/apisix-2.1-0.rockspec [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    "lua-resty-cookie = 0.1.0",
+    "lua-resty-session = 2.24",
+    "opentracing-openresty = 0.1",
+    "lua-resty-radixtree = 2.4",
+    "lua-protobuf = 0.3.1",
+    "lua-resty-openidc = 1.7.2-1",
+    "luafilesystem = 1.7.0-2",
+    "lua-tinyyaml = 1.0",
+    "lua-resty-prometheus = 1.1",
+    "jsonschema = 0.8",
```
Comment (membphis) [inline rockspec/apisix-2.1-0.rockspec:50]: this is wrong, please make a check:

the correct one: https://github.com/apache/apisix/blob/master/rockspec/apisix-master-0.rockspec#L50

**Review by membphis — CHANGES_REQUESTED** [review]
CI failed, need to fix it too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +33,33 @@
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 2.1.0
+
+### Core
+- :sunrise: **support ENV variable in configuration.** [#2743](https://github.com/apache/apisix/pull/2743)
+- :sunrise: **support TLS connection with etcd.** [#2548](https://github.com/apache/apisix/pull/2548)
+- generate create/update_time automatically. [#2740](https://github.com/apache/apisix/pull/2740)
+- add a deprecate log for enable_websocket in upstream.[#2691](https://github.com/apache/apisix/pull/2691)
```
Comment (spacewander) [inline CHANGELOG.md:42]: Better to mention that `consumer.id` is deprecated too.

File: rockspec/apisix-2.1-0.rockspec (hunk 1 above)
Comment (moonming) [reply rockspec/apisix-2.1-0.rockspec]: fixed

**Comment by moonming** [comment]
added

Thanks,
Ming Wen
Twitter: _WenMing


罗泽轩 <notifications@github.com> 于2020年11月29日周日 上午11:43写道：

File: README.md [hunk 3]
```diff
@@ -167,10 +165,9 @@ There are several ways to install the Apache Release version of APISIX:
    - Installation runtime dependencies: OpenResty and etcd, and compilation dependencies: luarocks. Refer to [install dependencies documentation](doc/install-dependencies.md)
    - Download the latest source code release package:
      ```shell
-     $ mkdir apisix-2.0
-     $ cd apisix-2.0
-     $ wget https://downloads.apache.org/apisix/2.0/apache-apisix-2.0-src.tgz
-     $ tar zxvf apache-apisix-2.0-src.tgz
+     $ mkdir apisix-2.1
```
Comment (membphis) [inline README.md:168]: missing `cd apisix-2.1` ?

**Comment by moonming** [comment]
ping @membphis

**Comment by moonming** [comment]
Release vote for Apache APISIX 2.1 has PASSED and I will merge to master.

**Review by spacewander — APPROVED** [review]

**Comment by membphis** [comment]
@moonming create `2.1` tag, then I'll make a test to release new version of luarocks

**Review by agile6v — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**